Makefile

```diff
@@ -92,7 +92,8 @@ ldflags = -X github.com/cosmos/cosmos-sdk/version.Name=regen \
 	-X github.com/cosmos/cosmos-sdk/version.AppName=regen \
 	-X github.com/cosmos/cosmos-sdk/version.Version=$(VERSION) \
 	-X github.com/cosmos/cosmos-sdk/version.Commit=$(COMMIT) \
-	-X github.com/cosmos/cosmos-sdk/version.BuildTags=$(build_tags_comma_sep)
+	-X github.com/cosmos/cosmos-sdk/version.BuildTags=$(build_tags_comma_sep) \
+	-X google.golang.org/protobuf/reflect/protoregistry.conflictPolicy=warn
```

Comment:
I'm getting proto namespace-conflict error, not sure how to fix this.

```
WARNING: proto: file "cosmos/base/query/v1beta1/pagination.proto" is already registered
        previously from: "cosmossdk.io/api/cosmos/base/query/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/query/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: file "cosmos/base/query/v1beta1/pagination.proto" has a name conflict over cosmos.base.query.v1beta1.PageResponse
        previously from: "cosmossdk.io/api/cosmos/base/query/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/query/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: file "cosmos/base/query/v1beta1/pagination.proto" has a name conflict over cosmos.base.query.v1beta1.PageRequest
        previously from: "cosmossdk.io/api/cosmos/base/query/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/query/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: message cosmos.base.query.v1beta1.PageRequest is already registered
        previously from: "cosmossdk.io/api/cosmos/base/query/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/query/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: message cosmos.base.query.v1beta1.PageResponse is already registered
        previously from: "cosmossdk.io/api/cosmos/base/query/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/query/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: file "cosmos/base/v1beta1/coin.proto" is already registered
        previously from: "cosmossdk.io/api/cosmos/base/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: file "cosmos/base/v1beta1/coin.proto" has a name conflict over cosmos.base.v1beta1.DecProto
        previously from: "cosmossdk.io/api/cosmos/base/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: file "cosmos/base/v1beta1/coin.proto" has a name conflict over cosmos.base.v1beta1.IntProto
        previously from: "cosmossdk.io/api/cosmos/base/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: file "cosmos/base/v1beta1/coin.proto" has a name conflict over cosmos.base.v1beta1.DecCoin
        previously from: "cosmossdk.io/api/cosmos/base/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: file "cosmos/base/v1beta1/coin.proto" has a name conflict over cosmos.base.v1beta1.Coin
        previously from: "cosmossdk.io/api/cosmos/base/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: message cosmos.base.v1beta1.Coin is already registered
        previously from: "cosmossdk.io/api/cosmos/base/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: message cosmos.base.v1beta1.DecCoin is already registered
        previously from: "cosmossdk.io/api/cosmos/base/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: message cosmos.base.v1beta1.IntProto is already registered
        previously from: "cosmossdk.io/api/cosmos/base/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

WARNING: proto: message cosmos.base.v1beta1.DecProto is already registered
        previously from: "cosmossdk.io/api/cosmos/base/v1beta1"
        currently from:  "github.com/cosmos/cosmos-sdk/api/cosmos/base/v1beta1"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

```

Reply:
Looks like issue is from the SDK side. I think we should remove [cosmossdk.io/api](https://github.com/cosmos/cosmos-sdk/blob/v0.46.0-rc3/go.mod#L6) import from the `go.mod`.

Reply:
resolved by https://github.com/cosmos/cosmos-sdk/pull/12719